index.bs

```diff
@@ -596,9 +596,10 @@ This Section defines a number of new set of <a>Constrainable Properties</a> for
   <dd>See <a>contrast</a> constrainable property.</dd>
 
   <dt><dfn dict-member for="MediaTrackConstraintSet"><code>pan</code></dfn></dt>
-  <dd>A value of true is normalized to a value of empty {{ConstrainDouble}}.
+  <dd>A value of true is normalized to a value of empty {{ConstrainDoubleRange}}.
       A value of false is normalized to a value of undefined.
-      See <a>pan</a> constrainable property.</dd>
+      If the value after normalization is a double value or a {{ConstrainDoubleRange}} value (whether empty or not), that implies a <a>pan capability</a> request which is ideal in basic constraints and exact <a>required</a> in advanced constraints.
```

Comment:
> Not happy with this sentence, because it doesn't appear to add anything normative that I can see (did you intend it to?) Please remove.

It should normatively state that `navigator.mediaDevices.getUserMedia({video: {pan: true}})` gives lower (better) fitness distance for pan capable cameras than for pan incapable camers and that `navigator.mediaDevices.getUserMedia({video: {height: 480, width: 640, advanced: [{height: 2160, pan: true, tilt: true, width: 3840}]}})` resolves either with a pan and tilt capable UHD video track or with a VGA video track without pan or tilt support.

Reply:
@eehakkin I don't think it accomplishes that, because a _"implies a pan capability request"_ is not a thing in the constraints algorithm. I think we need https://github.com/w3c/mediacapture-image/pull/257 to fix this.